includes/Core/Util/Input.php

```diff
@@ -0,0 +1,43 @@
+<?php
+/**
+ * Input abstraction class.
+ *
+ * @package   Google\Site_Kit\Core\Util
+ * @copyright 2019 Google LLC
+ * @license   https://www.apache.org/licenses/LICENSE-2.0 Apache License 2.0
+ * @link      https://sitekit.withgoogle.com
+ */
+
+namespace Google\Site_Kit\Core\Util;
+
+/**
+ * Class Input
+ *
+ * @since n.e.x.t
+ * @access private
+ * @ignore
+ */
+class Input {
+
+	/**
+	 * Gets a specific external variable by name and optionally filters it.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @link https://php.net/manual/en/function.filter-input.php
```

Comment:
See above, there should be an empty line below this one.

Reply:
Added!